src/five/formlib/tests/test_formlib.py

```diff
@@ -61,9 +61,56 @@ def test_get_widgets_for_schema_fields():
     """
 
 
+from uuid import uuid4
+
+from Testing.ZopeTestCase.zopedoctest.functional import http
+
+def http_request(url, form_parts=None, body=None, auth=None):
+    """perform HTTP request from given parameters.
+
+    The primary purpose of this auxiliary function is to compute
+    the `Content-Length` header. While Python 2's `cgi` module
+    seems resilient to a wrong `Content-Length`, Python 3's `cgi` fails
+    badly (at least for a two small value).
```

Comment:
Did you mean `too` instead of `two`?
```suggestion
    badly (at least for a too small value).
```